utilities/load_testing/justfile

```diff
@@ -3,13 +3,20 @@ set dotenv-load := true
 DOCKER_HOST := if os() == "macos" { "host.docker.internal" } else { "172.17.0.1" }
 
```

Comment:
It'd be good to have the default listing recipe here, as with our other justfiles:

```
# Show all available recipes
@_default:
    just --list --unsorted
```

Reply:
Good call, added in 501f19f.